packages/changeset-validations/src/helpers/validate-changeset.ts

```diff
@@ -4,6 +4,13 @@ import type { ErrorRecord, FormValidateCallback } from 'ember-headless-form';
 import type { EmberChangeset } from 'ember-changeset';
 import { assert } from '@ember/debug';
 
+/**
+ * Validation helper for integrating `ember-changeset` based validations into headless forms:
+ *
+ * - pass a changeset to the form's `@data`
+ * - pass this helper into the form's `@validate` hook
```

Comment:
Super nit, no need to change

Could this sentence be updated to align closer with the `yup` integration one to show an example of this:

```suggestion
 * - pass this helper into the form's `@validate` hook `@validate={{validateChangeset}}`
```